locales/de.json

```diff
@@ -2842,6 +2860,35 @@
 		"text": {
 			"descriptionTeams": "Arbeitsgemeinschaft, Schul-Band oder Klassensprecherversammlung? Lege ein Team an, um untereinander Material und Neuigkeiten auszutauschen - innerhalb deiner Schule oder schulübergreifend.",
 			"studentsCantCreateTeams": "Die Schule erlaubt das Anlegen von Teams durch Schüler in deiner Schule nicht. Wende dich bitte an einen Lehrer oder Administrator."
+		},
+		"topics": {
+			"button": {
+				"addGroup": "Gruppe hinzufügen",
+				"addNewTool": "Neues Tool hinzufügen",
+				"addTopic": "Thema hinzufügen",
+				"deleteTopic": "Thema löschen",
+				"displayTopicAgain": "Thema wieder anzeigen",
+				"hideTopic": "Thema verstecken",
+				"importTopic": "Thema importieren",
+				"moveTopic": "Thema verschieben",
+				"shareTopic": "Thema teilen"
+			},
+			"headline": {
+				"collapse/ExpandMyTasks": "Meine Aufgaben ein/ausklappen",
+				"collapse/ExpandStudentGroups": "Schülergruppen ein/ausklappen",
+				"homework": "({{homeworkLength}} Aufgabe{{#ifneq 1 {{homeworkLength}}}}n{{/ifneq}})",
```

Comment:
did you test if this works?

Reply:
The Teams sidebar didn't show in the localhost? can't check properly.  It supposed to work. may be others need to look at this.